src/pages/blog/2024-06-28-why-i-like-graphql.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: "Why, after 8 years, I still like GraphQL"
+tags: ["in-production"]
+date: 2024-06-28
+byline: Marc-André Giroux, edited by Jeff Auriemma
+---
+
+This post is edited from its [original publication](https://www.magiroux.com/eight-years-of-graphql/) on Marc-André Giroux's blog.  Though originally intended to respond to a specific article, we at the GraphQL Foundation felt that Marc-André's post would stand well on its own with a few light edits.  What follows is a well-informed set of practices and principles that engineers should consider when using GraphQL in production.
```

Comment:
We need to be careful to indicate this is opinion, for example Marc-Andre not picking GraphQL for public APIs doesn't mean that no-one should - I for one much prefer GitHub's GraphQL API over their REST API.

I've also made it italic to make it clearer it's effectively an "editor's note"

```suggestion
_This is a guest post outlining GraphQL TSC emeritus Marc-André Giroux's thoughts on practices and principles engineers should consider when using GraphQL in production. Though the [original publication](https://www.magiroux.com/eight-years-of-graphql/) was intended to respond to a specific article, we at the GraphQL Foundation felt that Marc-André's well-informed opinions would stand well on their own with a little light editing._
```